### DISCUSSION TITLE
自动肉鸽烧水优先选2希望

### DISCUSSION THREAD

**Discussion by Hanch4n** [MaaAssistantArknights/MaaAssistantArknights#8823] [Ideas]
最近小型的肉鸽比赛办的很多，在这个过程中发现对于职业队玩家来说，2希望的优先级要远大于热水壶，而现在的随机直升机制导致职业队几乎不可能在短时间内同时凹出2希望+开局干员直升；而现在的烧水功能却导致能顺利凹出如上开局的成功率很低。希望可以在烧水功能里加一个优先选择2希望的选项

**Comment by langkenan233** [comment]
附议
